lib/membrane_rtmp_plugin/rtmp/source/message_handler.ex

```diff
@@ -293,4 +297,8 @@ defmodule Membrane.RTMP.MessageHandler do
         parse_packet_messages(<<>>, message_parser, [{nil, step} | messages])
     end
   end
+
+  @compile {:inline, socket_module: 1}
```

Comment:
I wonder if we really need this little bit of performance here, I leave this decision up to you